subset/cloud/test_udmi

```diff
@@ -5,13 +5,20 @@ source reporting.sh
 REPORT=/tmp/report.txt
 LOG=/tmp/udmi.log
 
+function delay_finish {
+    # Travis doesn't always wait for buffer to flush on exit, so give some time.
```

Comment:
nit: no more travis. Not sure if this is even necessary anymore, but doesn't hurt.